handbook/engineering/security/index.md

```diff
@@ -4,9 +4,51 @@ We think that security is an enabler for the business. Sourcegraph is committed
 
 ## Goals
 
-Customers know they can confidently and securely use Sourcegraph with their private code.
+### Long term
 
-[This (internal) document](https://docs.google.com/document/d/1HhYfnkC17yCPqqw91H1QAF7q8toOOiNtoOOw06hqrNQ/) details our plan.
+Every organization, is confident they can trust Sourcegraph with the security of their Source code.
+
+### Medium term
+
+The goals below lie on the path to our long-term goals, and represent the next 3-6 months of work.  These goals will be delivered serially, but may be worked on in parallel.
+
+**We share our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph cloud**
+
+  - Problem and rationale: While Sourcegraph cloud is in an good, and improvine security state, we currently do not have the right level of visibility. By starting with a private repository that contains no secrets, we can build and deploy the visibility technologies and processes that allow us to build a holistic view of the environment.  This serves as a proxy for small company repository use.
+  - Planned work:
+    - Run ongoing vulnerability scans of the containers including vulnerability remediation
+    - Deploy centralized security logging
+    - Ship container logs to the logging destination
+    - Begin documenting policies
+  - Definition of success: Frequent container vulnerability scans are being run, and stored for future a audit. Fixes to the containers or their underlying service configurations have made it into the product release cycle.  Logs of new container deployments are available for search in a centralized logging tool.
+
+**We share a private repository containing encrypted secrets**
+
+  - Problem and rationale: Customers will have repositories containing encrypted secrets, using an [ansible vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) or gpg encrypted files. Trusting these repositories to external systems requires the added degree of care and concern, expected in security conscious small and medium enterprise (SME). To support those needs, not only is enhanced visibility into our own systems required on our side, it will be contractually obligated, or at a minimum part of security questionnaires.
```

Comment:
This wording makes it seam like we will be encrypting customer secrets with either of these solutions. Is that the case?   Could we phrase this as a problem to be solved rather than a solution?

Reply:
Where did you view this as Sourcegraph encrypting customer secrets? I can definitely see changing the first sentence - and have pushed up a change that I believe addresses that.

Reply:
> Customers will have repositories containing encrypted secrets, using an [ansible vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) or gpg encrypted files.

"Will have" makes it seam like we're going to do this.

Reply:
Also updated in that same push. Text now shows as *Customers repositories can contain encrypted secrets, such as an [ansible vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html) or gpg encrypted files.*